charts/redis-18.19.2/templates/replicas/pvcs.yaml

```
@@ -0,0 +1,43 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if and (eq .Values.architecture "replication") (not .Values.sentinel.enabled) }}
+{{- if and (eq .Values.replica.kind "StatefulSet") .Values.replica.persistence.enabled (not .Values.replica.persistence.existingClaim) }}
+{{- $statefulsetName := printf "%s-replicas" (include "common.names.fullname" .) }}
+{{- $existingStatefulSet := lookup "apps/v1" "StatefulSet" (include "common.names.namespace" .) $statefulsetName }}
+{{- if $existingStatefulSet }}
+{{- $fullname := include "common.names.fullname" . }}
+{{- $claimLabels := include "common.tplvalues.merge" ( dict "values" ( list .Values.master.persistence.labels .Values.commonLabels ) "context" . ) }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)